koji/vars/external_repos.yml

```diff
@@ -20,17 +20,17 @@ external_repos:
     url: http://mirror.centos.org/centos/7/updates/x86_64/
   # EL8 repositories
   - name: centos8-baseos
-    url: http://mirror.centos.org/centos/8/BaseOS/$arch/os/
+    url: http://vault.centos.org/centos/8/BaseOS/$arch/os/
   - name: centos8-appstream
-    url: http://mirror.centos.org/centos/8/AppStream/$arch/os/
+    url: http://vault.centos.org/centos/8/AppStream/$arch/os/
   - name: test-flat-el8
     url: http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001/
   - name: centos8-devel
     url: https://vault.centos.org/8.4.2105/Devel/$arch/os/
```

Comment:
Now I wonder if we're even using this. It's quite old.

Reply:
Based on what I remember at the time, we had a dependency coming from this repo that got removed in 8.5+.  This repo was more of a catch-all of sorts by the Centos team for needed packages.  I don't remember which package at this time.